katsdpcontroller/web.py

```diff
@@ -103,6 +103,20 @@ def _rotate_handler(request: web.Request) -> dict:
     return {}
 
 
+async def _block_dashboard(request: web.Request) -> web.Response:
+    """Make dashboard wait for a response.
+
+    This seems to be the easiest way to trick it into thinking it's not
+    connected. We just need to put it to sleep for longer than the
+    refresh interval.
+    """
+    await asyncio.sleep(2)
+    response = web.json_response({}, status=404)
+    # Avoid spamming the logs many times a second
+    response['log_level'] = logging.DEBUG
```

Comment:
Confused :) - how does setting the log level to DEBUG help the log spamming ?

Reply:
It's the log level at which the details of the request are logged. And since only messages at INFO or higher actually end up in Logtrail etc, it will be invisible.

Reply:
So works in opposition to what you might expect when setting the log_level on an object